### ISSUE TITLE
Cannot build Wordgrinder 0.8

### ISSUE THREAD

**Issue by juppes2** [davidgiven/wordgrinder#192]
Hello,
I am on Linux Mint 20 Cinnamon using guake as my standard terminal emulator. I seem to have all the dependencies installed, but when I cd into the wordgrinder-0.8 folder and do 'make', I get this error:

make: *** Keine Regel vorhanden, um das Ziel „src/c/emu/lua-5.1.5/*.[ch]“, 
  benötigt von „.obj/lua“, zu erstellen.  Schluss.

Sorry, this German, but I hope it is understandable somehow. It's complaining about a missing rule, so „src/c/emu/lua-5.1.5/*.[ch]", necessary for "obj/lua" can be built -my translation.
What does this mean, and is there a fix/workaround for it? Wordgrinder 0.7.2 is running perfectly on my sytem...
I'd appreciate your help.

**Comment by davidgiven** [comment]
Did you by any chance download the 'minimal dependencies for Debian' version? That's missing half the files and is intended for packaging only.

Also, you might want to consider the latest development build --- 0.8 is quite old now.

**Comment by juppes2** [comment]
Yes, it is the minimal dependencies one indeed! But where can I get the latest development release? Whenever I click a link to the latest release, I end up with version 0.8. Of course I would like to try out the development release...
Thanks for providing a link - I may constantly be overlooking something ;-)

**Comment by davidgiven** [comment]
If you did `sudo make install`, then it's likely that it's been installed into _root_'s home directory, which is probably not what you want! Try just `make install`?

**Comment by juppes2** [comment]
Thanks for your reply. I looked through my root folder, but no trace of wordgrinder. And another strange thing about version 0.8 (not the function, but the file placement after installation): I did 'sudo make install PREFIX=/usr/bin'. But afterwards the wordgrinder and xwordgrinder files end up in /home/USER/bin with owner root. Invoking 'wordgrinder' from terminal without sudo just launches wordgrinder, and I can start typing... No admin password necessary.
